modis_lai_fluxnet_sites/create_artifacts.jl

```diff
@@ -0,0 +1,112 @@
+################################################################################
+# IMPORTS                                                                      #
+################################################################################
+
+using JSON
+
+using Formatting
+using Statistics
+using Dates
+using DataFrames
+using CSV
+using Downloads
+
+using ClimaArtifactsHelper
+
+
+################################################################################
+# CONSTANTS                                                                    #
+################################################################################
+
+OUTPUT_DIR = "modis_lai_fluxnet_sites_artifact"
+
+# Info for AMERIFLUX sites US-Ha1, US-MOz, US-NR1, US-Var
+# PRODUCT = "MCD15A2H"
+# BAND = "Lai_500m"
+# NETWORK = "AMERIFLUX"
+# START_DATE = "A20020704"
+# END_DATE = "A20250602"
+
+JSON_URLS = [
+    "https://modis.ornl.gov/rst/api/v1/MCD15A2H/AMERIFLUX/US-Ha1/subset?band=Lai_500m&startDate=A20020704&endDate=A20250602",
+    "https://modis.ornl.gov/rst/api/v1/MCD15A2H/AMERIFLUX/US-MOz/subset?band=Lai_500m&startDate=A20020704&endDate=A20250602",
+    "https://modis.ornl.gov/rst/api/v1/MCD15A2H/AMERIFLUX/US-NR1/subset?band=Lai_500m&startDate=A20020704&endDate=A20250602",
+    "https://modis.ornl.gov/rst/api/v1/MCD15A2H/AMERIFLUX/US-Var/subset?band=Lai_500m&startDate=A20020704&endDate=A20250602",
+]
+
+MODIS_FILE_PATHS = [
+    "modis_lai_US-Ha1.csv",
+    "modis_lai_US-MOz.csv",
+    "modis_lai_US-NR1.csv",
+    "modis_lai_US-Var.csv",
+]
+
+SITES = [
+    "US-Ha1",
+    "US-MOz",
+    "US-NR1",
+    "US-Var",
+]
+
+################################################################################
+# HELPERS                                                                      #
+################################################################################
+
+function single_col_data_matrix(JSON_data::Vector)
+    """Takes in a vector of JSON data dicitonaries of a single column of chunked
+    MODIS data and assembles it into a data matrix with a time column and a
+    single data column. The data column gives the average of the grid cell 
+    arround the site for each day, and the time column gives the Dates.DateTime 
+    object for each day. In averaging, MODIS data marked as missing (>100) is 
+    ignored."""
```

Comment:
Thank you for the helpful comment! Docstrings in Julia are usually written like this (which helps with auto-generating documentation):
```suggestion
"""
    single_col_data_matrix(JSON_data::Vector)

Takes in a vector of JSON data dicitonaries of a single column of chunked
MODIS data and assembles it into a data matrix with a time column and a
single data column. The data column gives the average of the grid cell 
arround the site for each day, and the time column gives the Dates.DateTime 
object for each day. In averaging, MODIS data marked as missing (>100) is 
ignored.
"""
function single_col_data_matrix(JSON_data::Vector)
```